### PR TITLE
Add ZzFX w/ git auto-update

### PR DESCRIPTION
**Pull request by jdnichollsc**
The purpose of this PR is adding [ZzFX](https://github.com/KilledByAPixel/ZzFX) to the cdnjs.

Thank you!

### CODE REVIEW

File: packages/z/ZzFX.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/KilledByAPixel/ZzFX.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "ZzFX.micro.js",
+          "ZzFX.js"
```
Comment (MattIPv4) [inline packages/z/ZzFX.json:34]:
```suggestion
          "ZzFX.*.js"
```
Let's just include all versions

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: packages/z/ZzFX.json (hunk 1 above)
Comment (jdnichollsc) [reply packages/z/ZzFX.json]: Done, thanks for your correction!

**Comment by jdnichollsc** [comment]
@MattIPv4 what do you think about the last change? Thanks!

**Comment by MattIPv4** [comment]
Will take a look at it when I have some time 👍

**Comment by jdnichollsc** [comment]
Thanks for your help! <3

File: packages/z/ZzFX.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/KilledByAPixel/ZzFX.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "ZzFX.*.js"
```
Comment (MattIPv4) [inline packages/z/ZzFX.json:33]:
```suggestion
          "ZzFX?(.*).js"
```
Make extra period + part optional

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

**Review by MattIPv4 — APPROVED** [review]
Lgtm

**Pull request merged**